kubernetes/api/CoreV1API.c

```diff
@@ -35193,9 +35193,11 @@ CoreV1API_readNamespacedPodLog(apiClient_t *apiClient, char *name, char *_namesp
     //    printf("%s\n","Unauthorized");
     //}
     //primitive return type simple
-    char* elementToReturn =  strdup((char*)apiClient->dataReceived);
+    char* elementToReturn = NULL;
```

Comment:
This is a good catch !
Can you give a code change in the upstream code generator instead of here ?

https://github.com/OpenAPITools/openapi-generator/blob/ae9084bfd6bf03d3ef1fff1cdb52d04b008f9621/modules/openapi-generator/src/main/resources/C-libcurl/api-body.mustache#L362

Reply:
This PR is for your reference: https://github.com/OpenAPITools/openapi-generator/pull/17540